<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsViewSetting</class>
 <widget class="QDialog" name="GraphicsViewSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>286</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Map Widget Argument Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Distance Per Latitudem(m)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="xPerEdit">
          <property name="text">
           <string>14700</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Distance Per Lontitude(m)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="yPerEdit">
          <property name="text">
           <string>14700</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Warring Raudio(m)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="raudioEdit">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Latidute Top</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLineEdit" name="latTopEdit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Lonidute Top</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLineEdit" name="lonTopEdit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Latidute Button</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLineEdit" name="latBtmEdit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Lontidute Button</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLineEdit" name="lonBtmEdit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Level 2</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Level 3</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1" colspan="2">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Level 4</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1" colspan="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Level 5</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QLineEdit" name="viewLv3"/>
        </item>
        <item row="9" column="3">
         <widget class="QLineEdit" name="viewLv4"/>
        </item>
        <item row="10" column="3">
         <widget class="QLineEdit" name="viewLv5"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>View Rect Width(m)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Level 1</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QLineEdit" name="viewLv2"/>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="viewLv1"/>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Dead Time(ms)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLineEdit" name="deadTimeEdit">
          <property name="text">
           <string>1000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>66</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>467</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GraphicsViewSetting</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
